libs/blocks/global-navigation/blocks/profile/profile.css

```diff
@@ -0,0 +1,133 @@
+:root {
+  --border-color: #e1e1e1;
+}
+
+.gnav-profile .gnav-profile-menu {
```

Comment:
Same `.feds-` prefix comment for this file and all others, will stop adding this comment :)